src/debugger/scriptImporter.ts

```diff
@@ -86,4 +87,14 @@ export class ScriptImporter {
                 return new FileSystem().writeFile(sourceMappingLocalPath, updatedContent);
             });
     }
+
+    /**
+     * Changes the port of the url to be the one configured as this.packagerPort
+     */
+    private overridePackagerPort(urlToOverride: string): string {
```

Comment:
I agree it's not directly the responsibility of the ScriptImporter , but I don't think it's the appWorker responsibility either, and given that it's a much more complex class, I'd like to try to keep it as simple and thin as possible. Where do you think we should put this method?